instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py

```diff
@@ -284,6 +284,7 @@ def response_hook(span: Span, status: str, response_headers: List):
 _ENVIRON_ACTIVATION_KEY = "opentelemetry-flask.activation_key"
 _ENVIRON_REQCTX_REF_KEY = "opentelemetry-flask.reqctx_ref_key"
 _ENVIRON_TOKEN = "opentelemetry-flask.token"
+_ENVIRON_REQUEST_ROUTE_KEY = "request-route_key"
```

Comment:
Actually, is there any reason to use `wrapped_app_environ` rather than just a plain variable which is set in `_start_response` using `nonlocal`? Or maybe even just modifying `attributes` directly?

Reply:
I use `wrapped_app_environ` becouse is used above in the method. Do you see this as a problem? In mi point of view it is not a problem or bad decision to used it!

Reply:
I think the other cases need to use `wrapped_app_environ` because the value is accessed somewhere else. Here a local variable is an option and would be more efficient and probably simpler. But ultimately it's a small difference.